<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RatWalkGui::MdiMainWindow</class>
 <widget class="QMainWindow" name="RatWalkGui::MdiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>571</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>QtRatWalk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabsMovable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="videoSubWindow">
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="windowTitle">
        <string>Video</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QWidget" name="ratWalkFrame" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="pnlTools">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="checkBoxMostrarEsqueleto">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Mostrar esqueleto</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnStartStep">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Inicio Paso</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDiscardStep">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Descartar paso</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnFinishStep">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Fin Paso</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnEreaseStep">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Borrar Paso</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnTraerEsqueleto">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Traer esqueleto</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="horizontalSlider">
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="tracking">
              <bool>false</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="RatWalkGui::ImageViewer" name="pnlFrame" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btnPrev">
                <property name="text">
                 <string>Prev</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxCambiarFrame"/>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btnNext">
                <property name="text">
                 <string>Next</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="statusBarLayout"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="projectSubWindow">
       <property name="windowTitle">
        <string>Proyecto</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="twProjecto">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <column>
           <property name="text">
            <string>Projecto</string>
           </property>
           <property name="background">
            <color>
             <red>236</red>
             <green>236</green>
             <blue>236</blue>
            </color>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport_angles"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Ventana</string>
    </property>
    <widget class="QMenu" name="menuShow_angles">
     <property name="title">
      <string>Ángulos</string>
     </property>
    </widget>
    <addaction name="actionShow_projects"/>
    <addaction name="actionShow_video"/>
    <addaction name="actionMostrar_zoom"/>
    <addaction name="menuShow_angles"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_projects">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Mostrar proyectos</string>
   </property>
  </action>
  <action name="actionShow_video">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Mostrar video</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionDelete_point">
   <property name="text">
    <string>Borrar punto</string>
   </property>
  </action>
  <action name="actionExport_angles">
   <property name="text">
    <string>Exportar ángulos</string>
   </property>
  </action>
  <action name="actionMostrar_zoom">
   <property name="text">
    <string>Mostrar zoom</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RatWalkGui::ImageViewer</class>
   <extends>QWidget</extends>
   <header>RatWalkGui/ImageViewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
